www/js/controllers/search-controller.js

```diff
@@ -36,20 +36,13 @@ angular.module('pvta.controllers').controller('SearchController', function($scop
     var stops = [];
     if(StopList.isEmpty()){
       stops = Stops.query(function(){
```

Comment:
Good point, except when I ran it and went first to Search page and then to Stops, the Stops page had them loaded sorted by nearest. There must be something going on that the Search page isn't actually pushing the list it gets to StopList.